src/sentry/web/forms/accounts.py

```diff
@@ -358,3 +366,10 @@ def clean_password(self):
         password = self.cleaned_data["password"]
         password_validation.validate_password(password, user=self.user)
         return password
+
+    def clean_tos_check(self):
```

Comment:
I'm assuming this is opaque Django automagic, where methods that start with `clean_*` are called on each form field?

Reply:
yes, that is correct